Assignments/Sorting/SelectionSort.playground/Pages/Selection Sort.xcplaygroundpage/Contents.swift

```diff
@@ -9,19 +9,36 @@ Better define an inner function to find the next minimum in the array.
 
 */
 
-func selectionSort(array: [Int]) -> [Int] {
-    // You may declare array argument with var keyword so that it is copied.
-
-    return array
+func selectionSort(var array: [Int]) -> [Int] {
+  
+  let length = array.count
+  var indexOfMinimum: Int
+  
+  for var i = 0; i < (length - 1); i++ {
```

Comment:
Should check the guidelines we use https://github.com/raywenderlich/swift-style-guide#control-flow `for i in 0..<length - 1`